request/request-tests.ts

```diff
@@ -92,6 +92,7 @@ var options: request.Options = {
 	qs: obj,
 	json: value,
 	multipart: value,
+    agent: new http.Agent(),  
```

Comment:
It seems that this file is using tabs, but for your changes you used spaces.  As you can see in the diff things don't line up.  I know it's small, but if you could keep the whitespace consistent within this file that would be great.  Same with line 89 of `request.d.ts`.